src/brain/issueLabelHandler/followups.ts

```diff
@@ -26,10 +25,7 @@ import {
 } from '@utils/githubEventHelpers';
 import { isFromABot } from '@utils/isFromABot';
 
-const REPOS_TO_TRACK_FOR_FOLLOWUPS = new Set([
-  ...SENTRY_SDK_REPOS,
-  ...SENTRY_MONOREPOS,
-]);
+const REPOS_TO_TRACK_FOR_FOLLOWUPS = new Set(SENTRY_REPOS);
```

Comment:
Is there a good reason to keep `REPOS_TO_TRACK_FOR_FOLLOWUPS` around vs. referencing `SENTRY_REPOS` directly like we do for the stalebot case?

Reply:
I think it's mostly for the performance, looking up an item in a set is O(1) vs in a list which is O(n)

Reply:
And since this is declared at the top, the set is only created once. So it would need to be here so it's a bit more performant

Reply:
Is there anywhere where this will matter in practice? For anything we're practically doing, I imagine a list of like 40 (<- is that right in order of magnitude terms) items is about as fast to scan as a similarly sized set. I do find longer, more descriptive names better for maintainability - maybe we just pick one of these (I vote `REPOS_TO_TRACK_FOR_FOLLOWUPS`) and keep that as the name everywhere?

Reply:
No harm in declaring `SENTRY_REPOS` as a set actually in config, I'll do that. Unfortunately `REPOS_TO_TRACK_FOR_FOLLOWUPS` won't work because we use the same set of repos for determining which repos to enable stalebot for